modules/architecture-machine-roles.adoc

```diff
@@ -44,12 +44,19 @@ control the Kubernetes cluster. In {product-title}, the master machines are
 the control plane. They contain more
 than just the Kubernetes services for managing the {product-title} cluster.
 Because all of the machines with the control plane role are master machines,
-and the terms "master" and "control plane" are used interchangeably to describe
+the terms _master_ and _control plane_ are used interchangeably to describe
 them. Instead of being grouped into a
 MachineSet, master machines are defined by a series of standalone machine API
 resources. Extra controls apply to master machines to prevent you from deleting
 all master machines and breaking your cluster.
 
+[NOTE]
+====
+It is recommended to only use three master nodes. Although you can theoretically
```

Comment:
> It is recommended to only use three master nodes
Adding `only` is not suggested. Better to remove the word `only`, because users may want to create large size clusters.

Reply:
Makes sense; I'll remove it